v1/translate.yaml

```diff
@@ -0,0 +1,254 @@
+info:
+  version: '1.0.0'
+  title: Content Translation API
+  description: Content Translation API
+  termsofservice: https://github.com/wikimedia/restbase
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /machine/{from}/{to}{/provider}:
+    post:
+      tags: ['Translate']
```

Comment:
@Nikerabbit The tags are used to group the endpoints in the documentation page. See https://en.wikipedia.org/api/rest_v1/ - the endpoints are grouped by tag. So introducing a `translate` tag we will add a new top-level section and group all of these endpoints together int he doc.